src/makeRobot.js

```diff
@@ -38,7 +38,68 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      const id = this.name;
+      const chip = this.version;
+      const wheel = this.wheels;
+
+      return `name: ${id}, chip version: ${chip}, wheels: ${wheel}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goRight(n = 1) {
+      if (n > 0) {
+        this.coords.x += n;
+      }
```

Comment:
```suggestion
    goRight(step = 1) {
      if (step > 0) {
        this.coords.x += step;
      }
```
В других методах аналогично